modules/core/react/index.ts

```diff
@@ -33,11 +30,9 @@ export {
   colors,
   depth,
   spacing,
-  space,
-  SpaceProps,
   spacingNumbers,
+  stripUnit,
```

Comment:
@anicholls @lychyi you guys think we should move this into `common/lib/utils/stripUnit.ts`?

Reply:
I think it's fine as is. Otherwise we'll have to add `common` as a dep for `core` right? I think it can be refactored if other packages need it (outside of labs).